app/tests/unit/services/version.spec.js

```diff
@@ -208,3 +229,34 @@ describe('update', () => {
     expect(versionTrx.commit).toHaveBeenCalledTimes(1);
   });
 });
+
+describe('updateIsLatest', () => {
+  it('Updates a version of an object', async () => {
```

Comment:
This single test isn't sufficient to cover all potential branching cases. Line 310 branch isn't being fully covered: `current.isLatest` isn't being checked in the negative case.

Reply:
Acceptance criteria for the ticket was limited to `versionsService.updateIsLatest()`, `utils.getUniqueObjects()`, and `syncService.syncVersions()`. Should I expand the scope of the tests?